spec/controllers/downloads_controller_spec.rb

```diff
@@ -44,14 +44,19 @@
 
       context "with an alternative file" do
         context "that is persisted" do
+          let(:file) { File.open(fixture_file_path('world.png'), 'rb') }
+
+          # rubocop:disable Style/Semicolon
```

Comment:
I'd prefer being explicit about this by adding 'spec/**/*' as an exclude to the Style/Semicolon cop in .rubocop.yml than littering the codebase potentially with a bunch of one-line disable/enables. That way our ruleset explicitly says, "we don't care about semicolon usage in our test suite."